doc/state-snapshot.md

```diff
@@ -8,7 +8,7 @@ needs to be synced from scatch.
 To overcome these issues, we are providing a `cardano-db-sync` state snapshot, which should
 drastically reduce the time required to get `db-sync` back up and running after the database is
 dropped and recreated. This snapshot is compatible with both `cardano-db-sync` and
-`cardano-db-sync-extended` (which maintains an extra `epoch` table).
+`cardano-db-sync` with --no-epoch-table (which maintains an extra `epoch` table).
```

Comment:
What's the expected behaviour here? Is the Epoch table enabled and maintained by default?
The comment doesn't match the flag name's implied behaviour.